pkgs/tools/archivers/wiseunpacker/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, pkgs
+}:
+let
+  version = "1.3.3";
```

Comment:
Latest release is 1.4.2, but seems to be broken. This release works fine for me.

Reply:
Hi, original dev here. Nothing was ever reported to me about anything being broken. Please open an issue and describe what you're running into.

Reply:
I upgraded it locally to 1.4.2 and ran fetch-deps, and it builds okay.  There are no tests here though.  @Gigahawk do you remember what was wrong with it?

Reply:
https://github.com/mnadareski/WiseUnpacker/issues/5